Doc/library/asyncio-api-index.rst

```diff
@@ -48,6 +48,12 @@ await on multiple things with timeouts.
     * - :class:`Task`
       - Task object.
 
+    * - :class:`TaskGroup`
+      - Holds a group of tasks.
+
+    * - :class:`Runner`
+      - Manages multiple function calls.
+
```

Comment:
I'd recommend re-ordering these entries and rewording them eg:

```rst
Utilities to run asyncio programs, create Tasks, and
await on multiple things with timeouts.

.. list-table::
    :widths: 50 50
    :class: full-width-table

    * - :func:`run`
      - Create an event loop, run one async function, and close the loop.

    * - :class:`Runner`
      - A context manager to create an event loop, run one or more async functions, and close the loop.

    * - :class:`TaskGroup`
      - Combines a task creation API with a convenient and reliable way to wait for all tasks in the group to finish.

    * - :func:`create_task`
      - Start an asyncio Task.
```

Reply:
specifically `Runner` should be introduced at the same time as `run` and `TaskGroup` should be introduced before `create_task`